grammatical changes to resource metrics pipeline (#23128)
kubernetes/website: content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md

<!-- overview -->

Resource usage metrics, such as container CPU and memory usage,
are available in Kubernetes through the Metrics API. These metrics can be accessed either directly

> by the user, i.e. by using the `kubectl top` command, or by a controller in the cluster, e.g.

Reviewer: Avoid [Latin phrases](https://kubernetes.io/docs/contribute/style/style-guide/#avoid-latin-phrases):
```suggestion
by the user with the `kubectl top` command, or by a controller in the cluster, for example
```
